openml/datasets/functions.py

```diff
@@ -897,6 +901,46 @@ def edit_dataset(
     return int(data_id)
 
 
+def fork_dataset(data_id: int) -> int:
+    """
+     Creates a new dataset version, with possibly a different owner.
+     The authenticated user becomes the new owner.
```

Comment:
Merge two sentences to "Creates a new dataset version, with the authenticated user as the new owner."